src/Faithlife.Data/AsyncScope.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Threading.Tasks;
+
+namespace Faithlife.Data
+{
+	internal readonly struct AsyncScope : IAsyncDisposable
```

Comment:
Would there be any value in `AsyncScope<T> where T : IDisposable` and exposing `public T Value { get; }`?

Might avoid some doubling of code below by allowing `commandScope.Value` to be used to access the scoped command.

Reply:
Since `.ConfigureAwait(false)` returns `ConfiguredAsyncDisposable`, I can't think of a way to avoid the doubling. Do you have an example?